### ISSUE TITLE
Peripheral scan returns no peripherals

### ISSUE THREAD

**Issue by drdaz** [l0gg3r/LGBluetooth#9]
This is bizarre. I've set up a test project which pretty much duplicates your example project. But peripheral scan never finds a peripheral.

I can't see I'm doing anything at all different to you... And your example project works just fine.
[Hopspots_Comms_LGBT.zip](https://github.com/l0gg3r/LGBluetooth/files/82785/Hopspots_Comms_LGBT.zip)

Any idea what's up here?

**Comment by drdaz** [comment]
So a little more testing seems to show that the first call always returns nothing, while subsequent calls find the peripherals that are present. Not in your example project though... that works.

**Comment by drdaz** [comment]
Right... so this is an initialisation thing it seems. Some time needs to pass between the first call of sharedInstance and the first write.

**Comment by calvingit** [comment]
yes.you have to access other methods after centralManagerDidUpdateState called.

**Comment by sureshjoshi** [comment]
This is a known issue with Core Bluetooth. The first call is basically useless.
